website/static/js/profile.js

```diff
@@ -577,6 +609,41 @@ var ListViewModel = function(ContentModel, urls, modes) {
         }
         return true;
     });
+    
+
+    self.institutionsEmpty = ko.computed(function() {
+
+            for (var i=0; i<self.contents().length; i++) {
+                if (self.contents()[i].institutionEmpty()) { 
+                    return true;
+                }
+            }
+ 
+        return false;
+                
+    });
+
+    
+    self.extraFieldsEmpty = ko.computed(function() {
+        var i;
+        if (urls.crud === '/api/v1/settings/jobs/') {
```

Comment:
These conditionals look like they're determining if `self` is bound to a "schools" or "jobs" list on the page.

If we have to have this sort of flow, let's keep it in one place. Please refactor these into a single function so they can be reused.

I'd like to see that function return constants intead of a string, but that's not in the style guide and I'd like @sloria's opinion:

```javascript
// (pseudocode)
var SCHOOLS = 1;
var JOBS = 2;

var myViewModel = function () {
    this.fieldType = ko.computed(function() {
        if (urls.crud.indexOf('jobs') !== -1) { return JOBS; }
        if (urls.crud.indexOf('schools') !== -1) { return SCHOOLS; }
    });
};